src/app/containers/Metadata/index.jsx

```diff
@@ -166,14 +147,13 @@ const MetadataContainer = ({ metadata, promo }) => {
       <LinkedData
         brandName={brandName}
         canonicalLink={canonicalNonUkLink}
-        firstPublished={timeFirstPublished}
-        lastUpdated={timeLastPublished}
         logoUrl={defaultImage}
         noBylinesPolicy={noBylinesPolicy}
         publishingPrinciples={publishingPrinciples}
         seoHeadline={getTitle(promo)}
         type={pathOr(null, [pageType, 'schemaOrg'], pageTypeMetadata)}
         about={aboutTagsContent(pathOr(null, ['tags', 'about'], metadata))}
+        pageSpecific={pageSpecificLinkedData}
       />
       <Metadata
```

Comment:
This is still passed props that arnt used, like `articleAuthor`

Reply:
Good spot